docs/reference/contrib/modeladmin/create_edit_delete_views.rst

```diff
@@ -150,3 +150,12 @@ See the following part of the docs to find out more:
 
 See the following part of the docs to find out more:
 :ref:`modeladmin_overriding_views`
+
+.. _modeladmin_exclude_fields:
+
+-----------------------------------
+``ModelAdmin.exclude_fields``
+-----------------------------------
+
+**Expected value**: An array of Model fields to be excluded from Create/Edit pages.
```

Comment:
```**Expected value**: A list or tuple of fields names to be excluded from Create/Edit pages.```

I'm not sure about how to write the other paragraph, the ones I have checked were like this:
```See the following part of the docs to find out more: Overriding templates``` 

Any suggestion ?